### ISSUE TITLE
Master entries

### ISSUE THREAD

**Issue by mmoreram** [elcodi/docs#44]
This issue is for all Components/Bundles documentation.

All package pairs will be explained as is shown here
- Component explained. We will not explain the classes not each implementation. This is not useful at all indeed, but maybe what is really valuable is to explain what the component is intended for, explaining maybe some classes and some workflows
- How this component is used in Elcodi environment (Bundle)
- How this component can be used outside Elcodi environment

---

@elcodi/deciders @cordoval @octante @hacfi @tonipinel @mickaelandrieu Once each component is implemented, please, edit the line with the link of the entry, as shown in this example. Then, any revision of any finished component, create new issues.
- [ ] [Sitemap](http://elcodi.io/docs/components/sitemap/)

Each component must be clearly documented, so ask for help if you need it :+1: 

---
- [x] Attribute
- [x] EntityTranslator
- [ ] Create a Geo center
- [ ] Create a media center
- [x] Menu
- [ ] Create a metric bucket
- [ ] Plugin engine
- [ ] How shipping works
- [ ] Manage your Sitemap files
- [ ] StateTransitionMachine
- [ ] Template engine
- [ ] Test Common
- [ ] Fixtures Booster

**Comment by mickaelandrieu** [comment]
I agree and I have started  the docs for Product. But it's a complicated component useful only with some others components (attribute, shipping, currency).

I'm not sure your components are really uncoupled the way they are builts and maybe some of them should be merged.

**Comment by mmoreram** [comment]
Just push whatever you have and we can work on top of your work. If we do things in a collaborative way, the result will be even better.

:)

**Comment by cordoval** [comment]
I cannot edit the checklist can you please provide proper permissions to do so?

**Comment by cordoval** [comment]
check the first one https://github.com/elcodi/docs/pull/51
